site/content/docs/5.1/getting-started/javascript.md

```diff
@@ -12,6 +12,18 @@ Plugins can be included individually (using Bootstrap's individual `js/dist/*.js
 
 If you use a bundler (Webpack, Rollup...), you can use `/js/dist/*.js` files which are UMD ready.
 
+## Usage with JavaScript frameworks
+
+{{< callout warning >}}
+While the Bootstrap CSS can be used with any framework, **the Bootstrap JavaScript is not fully compatible with frameworks like React, Vue, and Angular** which assume full knowledge of the DOM. Both Bootstrap and the framework may attempt to mutate the same DOM element, resulting in bugs like dropdowns that are stuck in the "open" position.
+{{< /callout >}}
+
+A better alternative for those using React and similar frameworks is to use a framework-specific package **instead of** the Bootstrap JavaScript. Here are some of the most popular options:
+
+- React: [react-bootstrap](https://react-bootstrap.github.io/) or [Reactstrap](https://reactstrap.github.io/)
```

Comment:
Hey there, React-bootstrap maintainer here.  I think it's a good idea to remove Reactstrap since that project has dropped off in terms of activity.  There were discussions in the past to unify the Bootstrap react community and use a single project so we don't fragment the user base.  React-bootstrap is much more active and has feature parity with many versions of Bootstrap